packages/algorand-js/src/lib/compare.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Description: Compare Uint8Arrays
+ * @param a Uint8Array
+ * @param b Uint8Array
+ */
+export function compareArray (a: Uint8Array, b: Uint8Array): Boolean {
```

Comment:
Why only `Uint8Array`? This means i can't compare two string arrays. Can we make this more general? (using `any[]`)

Reply:
because `(method) Op.assertBytes(b: unknown): Uint8Array
Argument of type 'Uint8Array' is not assignable to parameter of type 'any[]'`

Reply:
also using any[] is not good. what if you pass one array of numbers and one Uint8array ? The code will return undesired results.
If you want to compare string we can add `Uint8array | string[] `.

Reply:
how about just using `unknown[]`?

Reply:
OK. Then please change the function name to `compareUint8Array` / `compareBytes`. 
`compareArray` suggests we can compare any two arrays.

Reply:
unkown also gives same results.
`Argument of type 'Uint8Array' is not assignable to parameter of type 'unknown[]'.`

Reply:
```
function compareArray (a: unknown, b: unknown): Boolean {
  if (Array.isArray(a) && Array.isArray(b)) {
    return a.length === b.length &&
      a.every((v, i) => v === b[i]);
  }
  return false;
}
```
This works

Reply:
ok, should we ignore this case ? `what if you pass one array of numbers and one Uint8array ?`

Reply:
we won't be using this in our code, i think we can

Reply:
@amityadav0 i think just renaming `compareArray` would be better in this case. Somehow, `Array.isArray(a)` is not accepting `Uint8Array`. :/

Reply:
```
function compareArray (a: unknown, b: unknown): Boolean {
  if (Array.isArray(a) && Array.isArray(b)) {
    return a.length === b.length &&
      a.every((v, i) => v === b[i]);
  }
  return false;
}
```
this code doesn't work because, it is considering Uint8array as type, so it requires array  of Uint8array

Reply:
updated

Reply:
@ratik21 I have updated code to take Uin8array as well. let me know what do you think ?

Reply:
yes, i think we can use `instance of Uint8Array`. your call.